rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/180_update_dense_vector_type.yml

```diff
@@ -2118,21 +2126,15 @@ setup:
   - contains: {hits.hits: {_id: "31"}}
 
 ---
-"Test update flat --> bbq_flat --> bbq_hnsw":
+"Test update flat --> bbq_flat --> bbq_hnsw --> bbq_disk":
   - requires:
       capabilities:
         - method: POST
           path: /_search
-          capabilities: [ optimized_scalar_quantization_bbq ]
+          capabilities: [ update_field_to_bbq_disk, dense_vector_updatable_bbq ]
```

Comment:
I changed some capabilities, as they were referring to /_mapping instead of /_search - we were skipping some tests due to the capabilities not being present in _search.